src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/EventHandlers/ApprenticeshipConfirmationCommencedEventHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using MediatR;
+using NServiceBus;
+using SFA.DAS.CommitmentsV2.Application.Commands.ApprenticeshipConfirmationCommenced;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers
+{
+    public class ApprenticeshipConfirmationCommencedEventHandler : IHandleMessages<ApprenticeshipConfirmationCommencedEvent>
+    {
+        private readonly IMediator _mediator;
+
+        public ApprenticeshipConfirmationCommencedEventHandler(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        public Task Handle(ApprenticeshipConfirmationCommencedEvent message, IMessageHandlerContext context)
+        {
+            return _mediator.Send(new ApprenticeshipConfirmationCommencedCommand(message.CommitmentsApprenticeshipId,
+                message.CommitmentsApprovedOn, message.ApprenticeshipConfirmationOverdueOn));
+        }
+    }
+
+    public class ApprenticeshipConfirmationCommencedEvent
+    {
+        public Guid ApprenticeId {get; set;}
+        public long? ADCApprenticeshipId {get; set;}
```

Comment:
We have this left-over term "ADC" from some point during the re-re-naming of the Confirm Your Apprenticeship service, but I don't think it's really a domain term.

As there is the `CommitmentsApprenticeshipId`, and the other CS / ADC / CYA identifiers are not qualified, I don't think we should make a special case for the apprenticeship ID.

Although, I don't think this value is actually used by Commitments, so maybe it's a moot point at the moment.

```suggestion
        public long? ApprenticeshipId {get; set;}
```

Reply:
Yes, I took the name from Roy's document, but happy to use an agreed name.